src/components/implementation/no_interface/llbooter/llbooter.c

```diff
@@ -344,18 +429,64 @@ execute(void)
 	crt_compinit_execute(boot_comp_get);
 }
 
+void
+init_done_chkpt(struct crt_comp *c)
+{
+	struct crt_comp  *new_comp = boot_comp_get(crt_ncomp() + 1);
+	struct crt_chkpt *chkpt;
+	thdcap_t          thdcap;
+	int               ret;
+	char              name[32];
```

Comment:
Previously, component names were bounded by `INITARGS_MAX_PATHNAME`. Should keep that.